pandas/tests/extension/decimal/array.py

```diff
@@ -54,8 +53,8 @@ def __init__(self, values, copy=False):
         # self._values = self.values = self.data
 
     @classmethod
-    def _from_sequence(cls, scalars, copy=False):
-        return cls(scalars)
+    def _from_sequence(cls, scalars, dtype=None, copy=False):
+        return cls(scalars, copy=copy)
```

Comment:
minor nit: copy does not need to be passed here

Reply:
@jorisvandenbossche On your example above where you ask "Is this difference intended:", there is no way to avoid this.  If you do `np.array([1, 2, 3]) + a`, then the numpy `__add__()` method is called. If you did `pd.Series([1,2,3]) + a` in your example, it works, because then the `Series` method sees that the right operand is an EA, and dispatches to the reverse operator of the EA.